### PR TITLE
Default `pre-commit` hooks?

### PR DESCRIPTION
**Pull request by paddyroddy**
Let's give it a go?

### CODE REVIEW

**Comment by paddyroddy** [comment]
@p-j-smith need to work out how to autoupdate the local hooks!

File: renovate/default-config.json [hunk 1]
```diff
@@ -7,6 +7,7 @@
     ":enablePreCommit"
   ],
   "commitMessageAction": "Renovate:",
+  "reviewers": ["team:mirsg-rses"],
```
Comment (paddyroddy) [inline renovate/default-config.json:10]: Oops, meant to add this in #2!

**Review by p-j-smith — APPROVED** [review]
awesome!!

Is the mirsg precommit hook wrapper versioned? Is that something we need or it's better without a version?

**Comment by p-j-smith** [comment]
> @p-j-smith need to work out how to autoupdate the local hooks!

Is this because there's already a renovate config and you can't add a second? Maybe you need to move the mirsg pre commit hook into it's own repo and setup the renovate config there?

Ignore me if that's not what you meant!

**Comment by paddyroddy** [comment]
> Is this because there's already a renovate config and you can't add a second? Maybe you need to move the mirsg pre commit hook into it's own repo and setup the renovate config there?

So I originally called the file `precommit/mirsg-hooks.yaml` but then realised that the `renovate` `pre-commit` manager won't update the hooks, due to the fact that the file has to follow this format https://github.com/renovatebot/renovate/blob/b7d1f0f64fdec0b4fc08f14a6e9c158a0020e2b6/lib/modules/manager/pre-commit/index.ts#L13C16-L13C50.

**Comment by paddyroddy** [comment]
Turns out disabling tagging isn't possible, as we are all admins...

**Comment by paddyroddy** [comment]
image
Actually it might be possible!

**Pull request merged**